### PR TITLE
Added jenkins usecase, preparation for other CIs and fix some titles

### PR DESCRIPTION
**Pull request by MatanHeledPort**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

**Comment by MatanHeledPort** [comment]
We need illustrations for the use-cases @MPTG94 @Zohar2506

File: docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      - run: python get_port_entity.py
+
+workflows:
+  # ... other workflows
+  deploy-production-service:
+    jobs:
+      # ... other jobs
+      - report-to-port:
+        context:
+          - port
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md:72]: Close highlight under this line

File: docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+```
+
+</details>
+
+<br></br>
+
+Create the following Python script in your repository ot either create/update or get Port entities:
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md:80]:
```suggestion

```

File: docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+:::note
+In this example, we use Python modules which need to be installed. You can use the following `requirements.txt`:
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md:83]:
```suggestion
In the following example, we use Python modules which need to be installed. You can use the following `requirements.txt`:
```

File: docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+:::
+
+<Tabs groupId="usage" defaultValue="upsert" values={[
+{label: "Create/Update", value: "upsert"},
+{label: "Get", value: "get"}
+]}>
+
+<TabItem value="upsert">
+
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md:102]:
```suggestion

Create the following Python script in your repository to create or update Port entities as part of your pipeline:

```

File: docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+}
+
+# request url : {API_URL}/blueprints/<blueprint_id>/entities
+create_response = requests.post(f'{API_URL}/blueprints/test-blueprint/entities?upsert=true', json=entity_json, headers=headers)
+print(json.dumps(get_response.json(), indent=4))
+```
+
+</TabItem>
+<TabItem value="get">
+
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/circleci-workflow/circleci-workflow.md:141]:
```suggestion

Create the following Python script in your repository to get Port entities as part of your pipeline:

```

**Review by MPTG94 — APPROVED** [review]
LGTM, please apply fixes

**Review by MPTG94 — APPROVED** [review]
LGTM

**Pull request merged**